src/pages/login/login.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="main-container">
+  <div class="login-form-container">
+    <h2>~ Login to Revolution ~</h2>
+    <form [formGroup]="loginForm" (ngSubmit)="onSubmit()" class="login-form">
```

Comment:
Przy reaktywnych formularzach `(ngSubmit)=onSubmit()` jest handlerem `(click)` na przycisku formularza czyli w tym przypadku Twoim "Click to login"